warehouse/config.py

```diff
@@ -241,6 +241,12 @@ def configure(settings=None):
         coercer=int,
         default=100,
     )
+    maybe_set(settings, "tuf.database.url", "TUF_DATABASE_URL")
+    maybe_set(settings, "tuf.metadata.url", "TUF_METADATA_URL")
+    maybe_set(settings, "tuf.api.url", "TUF_API_URL")
```

Comment:
Would it make sense to rename this to `RSTUF_API_URL` since RSTUF is providing the API in here?